openpype/hosts/fusion/plugins/load/load_usd.py

```diff
@@ -0,0 +1,73 @@
+from openpype.pipeline import (
+    load,
+    get_representation_path,
+)
+from openpype.hosts.fusion.api import (
+    imprint_container,
+    get_current_comp,
+    comp_lock_and_undo_chunk
+)
+
+
+class FusionLoadAlembicMesh(load.LoaderPlugin):
+    """Load USD into Fusion
+
+    Support for USD was added since Fusion 18.5
```

Comment:
Is it possible to perform a check to see if we are on the correct versions and if not `pass` ?

Reply:
I suppose we could do `fusion.GetVersion()`:
```python

Py3> print(fusion.GetVersion())
# {1: 18, 2: 0, 3: 2, 4: 6, 'App': 'Fusion'}
```

As such we could maybe do:
```python
version = fusion.GetVersion()
major = version[1]
minor = version[2]
is_usd_supported = (major, minor) < (18, 5)

class FusionLoadUSD(load.LoaderPlugin):
    # ...
    # ...
    # do not allow the loader
    enabled = is_usd_supported
```